unpub/lib/src/mongo_store.dart

```diff
@@ -73,16 +73,26 @@ class MongoStore extends MetaStore {
 
   @override
   Future<UnpubQueryResult> queryPackages({
-    required size,
-    required page,
-    required sort,
+    size,
```

Comment:
4. size, page, and sort are made optional to support the new endpoint for getting all packages